lib/workers/branch/status-checks.ts

```diff
@@ -14,10 +14,7 @@ async function setStatusCheck(
     branchName,
     context
   );
-  // Check if state needs setting
-  if (existingState === state) {
-    logger.debug(`Status check ${context} is already up-to-date`);
-  } else {
+  if (existingState !== state) {
```

Comment:
Why change this?

Reply:
It's tricky enough to make test for this branch, but I'll do if this one is important. Or maybe, something like this would be enough in the function first line?
```
logger.debug(`setStatusCheck(branchName = ${branchName}, state = ${state}, ...)`);

Reply:
Never mind, it was not so hard :)